GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -892,44 +992,197 @@ internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog lo
+                                List<(EffectEvent, EffectEvent, float)> aoeToAoeMatches = MatchEffectToEffect(filteredBouncingOrbsAoEs, filteredBouncingOrbsAoEs);
+
+                                // Orb indicator near the swipe cone
+                                foreach (EffectEvent orb in clawVoidOrbs)
+                                {
+                                    (long start, long end) lifespan = orb.ComputeLifespan(log, 2080);
+                                    var circle = new CircleDecoration(25, lifespan, Colors.Black, 0.5, new PositionConnector(orb.Position));
+                                    replay.Decorations.Add(circle);
+                                }
+
+                                // Orb to AoE effects
+                                foreach ((EffectEvent aoe, EffectEvent orb, float distance) in orbToAoeMatches)
+                                {
+                                    // Add aoe
+                                    (long start, long end) lifespanAoE = (aoe.Time, aoe.Time + aoe.Duration);
+                                    replay.AddDecorationWithGrowing(new CircleDecoration(200, lifespanAoE, Colors.LightOrange, 0.2, new PositionConnector(aoe.Position)), lifespanAoE.end);
+                                    // Add projectile
+                                    replay.AddProjectile(orb.Position, aoe.Position, lifespanAoE, Colors.Black, 0.5);
+                                }
+
+                                // AoE to AoE effects
+                                foreach ((EffectEvent endingAoE, EffectEvent startingAoE, float distance) in aoeToAoeMatches)
+                                {
+                                    // Add aoe
+                                    (long start, long end) lifespanAoE = (endingAoE.Time, endingAoE.Time + endingAoE.Duration);
+                                    replay.AddDecorationWithGrowing(new CircleDecoration(200, lifespanAoE, Colors.LightOrange, 0.2, new PositionConnector(endingAoE.Position)), lifespanAoE.end);
+                                    // Add projectile - Starts when the previous AoE ends because it's bouncing
+                                    (long start, long end) lifespanAnimation = (startingAoE.Time + startingAoE.Duration, endingAoE.Time + endingAoE.Duration);
+                                    replay.AddProjectile(startingAoE.Position, endingAoE.Position, lifespanAnimation, Colors.Black, 0.5);
+                                }
+                            }
                         }
                     }
+
+                    // Generic Orange AoE - Used in multiple sections of the encounter - Used for the void orbs during the tail and claw slams
+                    log.CombatData.TryGetEffectEventsByGUID(EffectGUIDs.HarvestTempleScalableOrangeAoE, out IReadOnlyList<EffectEvent> genericOrangeAoE);
```

Comment:
Should be inside an if like the rest of TryGets, genericOrangeAoE list can be null.